### PR TITLE
Adding spatialpandas

### PR DESCRIPTION
**Pull request by AndrewAnnex**
recipe made by grayskull

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/spatialpandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - param
```
Comment (scopatz) [inline recipes/spatialpandas/meta.yaml:22]: Is param really needed at package build time?

Comment (AndrewAnnex) [reply recipes/spatialpandas/meta.yaml:22]: I am not the author of the project, they import it in their setup.py file for getting a formatted version number I think (see last failed CI build on this PR and their setup.py). Likely something that could be fixed in that project.

Comment (scopatz) [reply recipes/spatialpandas/meta.yaml:22]: I see OK

**Comment by AndrewAnnex** [comment]
Hey Anthony, that may have been premature as I hadn’t indicated yet that all maintainers agreed to be maintainers yet. I can remove the primary author of the project as a maintainer until we get a yes

-Andrew Annex

**Pull request merged**